lib/ansible/modules/windows/win_chocolatey_feature.ps1

```diff
@@ -0,0 +1,75 @@
+#!powershell
+# This file is part of Ansible
```

Comment:
I know this is a minor nitpick, but I am looking at making the Windows modules more consistent. Does the addition of `# This file is part of Ansible` has any useful meaning ?

Also, we tend to have `# -*- coding: utf-8 -*-` as the second line.

Reply:
BTW I see you are doing this for the .py file.

Reply:
The `# This file is part of Ansible` isn't needed IMO, I'm just copying pasting from another module and happy to have it removed. I don't think we should have `# -*- coding: utf-8 -*` as that is not a Windows/PowerShell-ism. There's no benefit for having that in the PowerShell modules.

Reply:
Alright, I always asumed `# -*- coding: utf-8 -*-` was an editor/IDE feature, but it does seem to be tight to Python specifically. Let me get rid of those then.

Reply:
#42853